media/html/visjsDebuggerPanel.html

```diff
@@ -48,6 +48,12 @@
           case 'stopRecordingVisjs':
             stopRecordingVisjs();
             break;
+          case 'exportAsPlantUml':
+          case 'exportAsGraphViz': {
+            const { fileName, file } = message;
+            downloadFile(fileName, file);
+            break;
+          }
```

Comment:
Shouldn't be necessary as mentioned [here](https://github.com/GinoCardillo-OST/Visual-OO-Debugger/pull/24/files#r772266710).

Reply:
Done, see [there](https://github.com/GinoCardillo-OST/Visual-OO-Debugger/pull/24/files#r772266710)